pulser-simulation/pulser_simulation/simulation.py

```diff
@@ -1156,6 +1156,8 @@ def from_sequence(
 class Simulation:
     r"""Simulation of a pulse sequence using QuTiP.
 
+    Note: This class is deprecated in favour of `QutipEmulator.from_sequence`.
+
```

Comment:
Do it like this so it appears properly formatted in the docs
```suggestion
    Note: 
        This class is deprecated in favour of `QutipEmulator.from_sequence`.

```